src/main/resources/stash-ssh-key-enforcer.properties

```diff
@@ -11,7 +11,7 @@ stash.ssh.reset.email.subject=Action Required: Reset Enterprise SSH Keys
 stash.ssh.reset.email.body=Your Key has been destroyed by the firey wrath of SSH Key Enforcer. Please generate a new key by visiting {0} 
 
 
-
+stashkeyenforcer.projectpermission.cancel=This Stash instance uses SSH Stash Key Enforcer. Users may not add keys directly to repositories or projects.
 
```

Comment:
Where is this being used? Is it just overriding the default message already being thrown?